addons/crm_survey/tests/test_crm_survey.py

```diff
@@ -0,0 +1,257 @@
+from odoo.addons.survey.tests import common
+from odoo.tests import tagged
+from odoo.tests.common import HttpCase
+from odoo.addons.mail.tests.common import MockEmail
+
+
+@tagged('-at_install', 'post_install', 'functional')
```

Comment:
why `functional`? Seems quite specific, usually, we only add tags if we have a good reason to do so :) But maybe I'm missing something though